.travis.yml

```diff
@@ -24,21 +24,21 @@ matrix:
     - language: python
       python: 2.7
       install:
-        - pip install paramiko pyyaml jinja2 python-etcd kubernetes GitPython
+        - pip install dnspython==1.13.0 paramiko pyyaml jinja2 python-etcd kubernetes GitPython
       script:
         - python -m unittest deployment.clusterObjectModel.test.test_cluster_object_model
```

Comment:
could you help to merge these tests like the other file?

Reply:
Done.